src/Microsoft.AspNetCore.Server.Kestrel/Http/FrameRequestStream.cs

```diff
@@ -165,13 +165,13 @@ private Task<int> ValidateState(CancellationToken cancellationToken)
                 case FrameStreamState.Open:
                     if (cancellationToken.IsCancellationRequested)
                     {
-                        return TaskUtilities.GetCancelledZeroTask();
+                        return TaskUtilities.GetCancelledZeroTask(cancellationToken);
                     }
                     break;
                 case FrameStreamState.Closed:
                     throw new ObjectDisposedException(nameof(FrameRequestStream));
                 case FrameStreamState.Aborted:
-                    return TaskUtilities.GetCancelledZeroTask();
+                    return TaskUtilities.GetCancelledZeroTask(default(CancellationToken));
```

Comment:
Unrelated to this change, but I noticed that `FrameResponseStream.ValidateState` only returns a canceled task for the `Aborted` case when `cancellationToken.IsCancellationRequested` is true, whereas here it is always returning a canceled task for `Aborted` regardless of `cancellationToken.IsCancellationRequested`.

`FrameResponseStream.ValidateState` ([src](https://github.com/aspnet/KestrelHttpServer/blob/b0d5cfcf47a8de83afe930056c87f193e4f955c6/src/Microsoft.AspNetCore.Server.Kestrel/Http/FrameResponseStream.cs#L160-L166)):

```csharp
                case FrameStreamState.Aborted:
                    if (cancellationToken.IsCancellationRequested)
                    {
                        // Aborted state only throws on write if cancellationToken requests it
                        return TaskUtilities.GetCancelledTask(cancellationToken);
                    }
                    break;
```

Is this by design, or should `FrameRequestStream` be modified to behave the same as `FrameResponseStream`?